ballerina/tests/xml_to_json_test.bal

```diff
@@ -224,119 +186,329 @@ function testElementWithDifferentNamedChildrenElementItems() {
     test:assertEquals(convertToJson("<root><hello attr0=\"hello\"></hello><hello></hello></root>"),
-    "{\"root\":{\"hello\":[{\"hello\":{\"@attr0\":\"hello\"}}, []]}}",
-    msg = "testElementWithSameNamedEmptyChildren result incorrect");
+                      "{\"root\":{\"hello\":[{\"@attr0\":\"hello\"}, \"\"]}}", msg =
+                      "testElementWithSameNamedEmptyChildren result incorrect");
 }
 
 @test:Config {
     groups: ["toJson"]
 }
 function testComplexXMLtoJson() {
-    test:assertEquals(convertToJson(
-    "<Invoice xmlns=\"example.com\" attr=\"attr-val\" xmlns:ns=\"ns.com\" ns:attr=\"ns-attr-val\">\n" +
-                            "        <PurchesedItems>\n" +
-                            "            <PLine><ItemCode>223345</ItemCode><Count>10</Count></PLine>\n" +
-                            "            <PLine><ItemCode>223300</ItemCode><Count>7</Count></PLine>\n" +
-                            "            <PLine><ItemCode discount=\"22%\">200777</ItemCode><Count>7</Count></PLine>\n" +
-                            "        </PurchesedItems>\n" +
-                            "        <Address xmlns=\"\">\n" +
-                            "            <StreetAddress>20, Palm grove, Colombo 3</StreetAddress>\n" +
-                            "            <City>Colombo</City>\n" +
-                            "            <Zip>00300</Zip>\n" +
-                            "            <Country>LK</Country>\n" +
-                            "        </Address>\n" +
-                            "    </Invoice>"),
-    "{\"Invoice\":{\"Invoice\":[\"\\n        \", {\"PurchesedItems\":[\"\\n            \", " +
-                            "{\"PLine\":[{\"ItemCode\":\"223345\", \"@xmlns\":\"example.com\"}, " +
-                            "{\"Count\":\"10\", \"@xmlns\":\"example.com\"}], \"@xmlns\":\"example.com\"}, " +
-                            "\"\\n            \", {\"PLine\":[{\"ItemCode\":\"223300\", \"@xmlns\":\"example.com\"}, " +
-                            "{\"Count\":\"7\", \"@xmlns\":\"example.com\"}], \"@xmlns\":\"example.com\"}, " +
-                            "\"\\n            \", {\"PLine\":[{\"ItemCode\":\"200777\", " +
-                            "\"@xmlns\":\"example.com\", \"@discount\":\"22%\"}, " +
-                            "{\"Count\":\"7\", \"@xmlns\":\"example.com\"}], \"@xmlns\":\"example.com\"}, " +
-                            "\"\\n        \"], \"@xmlns\":\"example.com\"}, \"\\n        \", " +
-                            "{\"Address\":[\"\\n            \", " +
-                            "{\"StreetAddress\":\"20, Palm grove, Colombo 3\"}, \"\\n            \", " +
-                            "{\"City\":\"Colombo\"}, \"\\n            \", {\"Zip\":\"00300\"}, \"\\n            \", " +
-                            "{\"Country\":\"LK\"}, \"\\n        \"], \"@xmlns\":\"\"}, \"\\n    \"], " +
+    test:assertEquals(convertToJson(e.toString()),"{\"Invoice\":{\"PurchesedItems\":{\"PLine\":[" +
```

Comment:
IINM `convertToJson` is a test util function. Why are there unit tests for this function?

Reply:
I had these as these test utils functions are keeping long times. But It seems these are not useful because the [toJson()](https://github.com/ballerina-platform/module-ballerina-xmldata/pull/291/files#diff-238e355567a10837bf03aed486fd6caac98a21b9d4e3b4403b86aec40e2d0012R77) directly calls the convertToJSON(). We can test this by calling ballerina API. So, we don't need to test the convertToJSON() by using these test utils. So I have removed this util module in 873cc8c.